### PR TITLE
logs-logs-logs: add more tests to better cover multiple characters case

### PR DESCRIPTION
**Pull request by Jasstkn**
Signed-off-by: Jasstkn <mariia.kotliarevskaia@gmail.com>

Add more tests to cover case that is described here https://github.com/exercism/go/issues/2232

With logic from issue:
```golang
type Applications struct {
	Key   rune
	Value string
}

//mapping based on struct to have an as defined ordered array
var mapping = []Applications{
	{ Key:   '❗', Value: "recommendation" },
	{Key:   '🔍', Value: "search"},
	{Key:   '☀',Value: "weather"},
}

// Application identifies the application emitting the given log.
func Application(log string) string {
	for _, val := range mapping {
		if strings.ContainsRune(log, val.Key) {
			return val.Value
		}
	}
	return "default"
}
```

With new tests it catches case where the first element in the variable `mapping` isn't the same as in the input log for `Application` function.
```golang
--- FAIL: TestApplication (0.00s)
    --- FAIL: TestApplication/multiple_characters_recommendation(2/3) (0.00s)
        logs_logs_logs_test.go:53: Application("🔍 search recommended product ❗") = "recommendation", want "search"
    --- FAIL: TestApplication/multiple_characters_recommendation(3/3) (0.00s)
        logs_logs_logs_test.go:53: Application("☀ weather is sunny ❗") = "recommendation", want "weather"
```

Also it fixes formatting that is described here: #2202

### CODE REVIEW

**Comment by Jasstkn** [comment]
hey @andrerfcsantos! thanks for providing me opportunity to contribute. I decided to add all of the three emoji in the tests so no matter what is the order in the mapping variable - we catch wrong behaviour. 
Looking forward to your feedback!

File: exercises/concept/logs-logs-logs/.docs/introduction.md [hunk 1]
```diff
@@ -38,31 +38,31 @@ Since `rune` is just an alias for `int32`, printing a rune's type will yield `in
 ```go
 myRune := '¿'
 fmt.Printf("myRune type: %T\n", myRune)
-// Output: myRune type: int32
+// => myRune type: int32
```
Comment (andrerfcsantos) [inline exercises/concept/logs-logs-logs/.docs/introduction.md:41]:
```suggestion
// Output: myRune type: int32
```

As discussed here https://github.com/exercism/go/issues/2202 when an instruction is a print statement, we actually want to keep the `Output:` wording.

Can you change this back in the other print statements?

**Review by andrerfcsantos — CHANGES_REQUESTED** [review]
Thanks for this!

Just left a minor comment.

File: exercises/concept/logs-logs-logs/.docs/introduction.md (hunk 1 above)
Comment (Jasstkn) [reply exercises/concept/logs-logs-logs/.docs/introduction.md]: Sorry, my bad. Reverted changes in https://github.com/exercism/go/pull/2236/commits/c773cd8c1bac7a02048db3e92e7abd3620ee32b6.

**Review by andrerfcsantos — APPROVED** [review]

**Pull request merged**